exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+There's a joke that goes:
+
+_There are 10 types of people in the world._
+_There are those who understand binary, and those who don't._
+
+You belong to the group that understands binary, and you and your binary-savvy friends want to create a secret handshake.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+So you come up with a sequence of actions, where each action corresponds to a different binary digit.
```

Comment:
I feel like binary is an implementation detail, not the story. And so I feel like the joke is maybe unnecessary and adds noise. So I suggest getting rid of that and focussing a little more on the fun story angle.

Suggestion:

```suggestion
You are making a secret coding club with some friends and friends-of-friends. 
You don't know everyone in the club, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
You don't want anyone who isn't in the know to be able to crack the code.

You and your friends have decided to come up with a code that use binary (because you're a geek and think binary is cool) and turns it into a series of actions that you do during the handshake.
```

Reply:
Nice, yeah. It's so funny how I even failed to consider that we could dump the binary thing.